testdata/bundles/registry/invalid/manifests/leader-election-role_rbac.authorization.k8s.io_v1_role.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Seems like a bunch of these manifests are irrelevant for the test of "AllNamespaces or not".

Can we get rid of everything except the CSV?

Reply:
Yup, good catch. That fine with me.

Reply:
Done :+1: